### DISCUSSION TITLE
Graduação EaD

### DISCUSSION THREAD

**Discussion by victorakio** [frontendbr/forum#1441] [Discussão]
Estou considerando fazer uma graduação em Sistemas de Informação EaD, estou na dúvida em qual faculdade fazer e não achei muitas opniões sobre a qualidade dos cursos sem ser no reclame aqui.

Alguém aí faz/já fez alguma graduação EaD? Se sim como foi sua experiência? Dava pra tirar dúvidas com os professores tinha um fórum para os alunos trocarem informações e etc?

**Comment by mateusmiguel** [comment]
Fala Victor, tudo bom? 

Faço ADS pela Anhembi Morumbi  / Laureate. Estou terminando o primeiro ano e tenho gostando bastante do material. Tem fóruns sim e com exceção de uma professora (de matemática) todos foram bem atenciosos em tirar dúvidas da turma. Ultimamente tenho notado uma melhora grande da plataforma, com mais vídeo aulas além da do material para leitura.

Qualquer dúvida, só chamar =]

**Comment by victorakio** [comment]
Ah legal @mateusmiguel, eu estava justamente interessado em fazer na Anhembi Morumbi mesmo.
Você já teve algum problema com os boletos? tinha bastante reclamação sobre isso no Reclame Aqui.

Valeu!
